### PR TITLE
PLT-3371 Change "system administrator" and "team administrator" to "System Administrator" and "Team Administrator" in UI text

### PR DESCRIPTION
**Pull request by chagara**
Made the changes as requested.

### CODE REVIEW

**Comment by enahum** [comment]
There's also the need to change the strings in `/webapp/i18n/en.json`

Thanks.

**Comment by jasonblais** [comment]
+1 for PM approval

**Comment by enahum** [comment]
We still need the change in `/webapp/i18n/en.json` before this can go in

**Comment by chagara** [comment]
Working on it. About to do a pull in a few minutes.

**Comment by chagara** [comment]
completed the changes as requested.

**Comment by enahum** [comment]
Ok two more things:

1. There are changes in the `en.json` file that haven't been made on the corresponding `jsx` files, so if you can please also change those.
2. This PR needs a rebase.

Thanks for taking the time to do this @chagara we highly appreciate your work

**Comment by chagara** [comment]
you mean the  i18n.jsx ? your welcome :) I'm glad to do this.

File: webapp/i18n/en.json [hunk 1]
```diff
@@ -188,7 +188,7 @@
   "admin.email.requireVerificationDescription": "Typically set to true in production. When true, Mattermost requires email verification after account creation prior to allowing login. Developers may set this field to false so skip sending verification emails for faster development.",
   "admin.email.requireVerificationTitle": "Require Email Verification: ",
   "admin.email.selfPush": "Manually enter Push Notification Service location",
-  "admin.email.smtpPasswordDescription": " Obtain this credential from administrator setting up your email server.",
+  "admin.email.smtpPasswordDescription": " Obtain this credential from Administrator setting up your email server.",
```
Comment (it33) [inline webapp/i18n/en.json]: Sorry, this is not a change requested in the ticket, and we leave "Administrator" as "administrator" if it's not the context of "Team Administrator" or "System Administrator"?

Comment (chagara) [inline webapp/i18n/en.json]: I apologize for that. I'll change it back.

**Comment by enahum** [comment]
not in i18n.jsx

What i mean is every string in the `en.json` file has a **key**, that **key** refers to an **id** in the code in the `*.jsx` files, so the thing is that the strings have to be changed to match the text in `en.json` wherever those **keys** are found.

**Comment by chagara** [comment]
I looked for all around and only 3 *.jsx files had the same string

**Comment by enahum** [comment]
Hi @chagara I found a few strings in jsx files that aren't updated with this PR

Here is a list for you:

File: `webapp/components/admin_console/privacy_settings.jsx`
String Ids: admin.privacy.showEmailDescription, admin.privacy.showFullNameDescription

File: `webapp/components/get_team_invite_link_modal.jsx`
String id: get_team_invite_link_modal.helpDisabled

File `webapp/components/tutorial/tutorial_tip.jsx`
String id: sidebar_header.tutorial

File: `webapp/components/signup_user_complete.jsx`
String ids: signup_user_completed.invalid_invite, signup_user_completed.no_open_server

**Comment by enahum** [comment]
this also needs a rebase

**Comment by enahum** [comment]
@chagara I'm going to close this PR, feel free to resubmit once the issues found are fixed.

Thanks again for your work, hopefully you'll submit this one again.

**Pull request closed without merging**